hypnotoad/cases/torpex.py

```diff
@@ -24,25 +24,27 @@
 
 from ..core.mesh import BoutMesh
 from ..core.equilibrium import (
-    setDefault,
     Equilibrium,
-    PsiContour,
     Point2D,
     EquilibriumRegion,
     SolutionError,
 )
 from ..geqdsk._geqdsk import read as geq_read
-from ..utils.hypnotoad_options import (
-    HypnotoadOptions,
-    optionsTableString,
-)
+from ..utils.options import WithMeta, is_positive, optionsTableString
 
 # type for manipulating information about magnetic field coils
 from collections import namedtuple
 
 Coil = namedtuple("Coil", "R, Z, I")
 
 
+def withDefault(option, default):
+    if option is not None:
+        return option
+    else:
+        return default
```

Comment:
I think `return option or default` would fail if `option` was supposed to be `0` or `False`. Could do something like `return default if option is None else option` if you prefer that?

Reply:
:facepalm: Yes, of course, sorry!

No, you've done it the more readable way. Only style suggestion would be to remove the `else:`

```suggestion
def withDefault(option, default):
    if option is not None:
        return option
    return default
```